main_test.go

```diff
@@ -287,8 +290,6 @@ func TestDefaultPort(t *testing.T) {
 }
 
 func TestPortOverride(t *testing.T) {
-	t.Parallel()
```

Comment:
why can't we run the parser tests in parallel?

Reply:
I disabled this while investigating and forgot the put it back. I added a commit that parallelizes (almost) all the tests